extensions/github-review-requests/src/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { Image, List, showToast, Toast } from "@raycast/api";
+import { useState } from "react";
+import { useDebounce } from "use-debounce";
+import { Actions } from "./actions";
+import { useSearch } from "./hooks/useSearch";
+
+export default function Command() {
+  const [searchText, setSearchText] = useState<string>();
+  const [debouncedSearchText] = useDebounce(searchText, 200);
+  const { result, loading, error } = useSearch(debouncedSearchText);
+
+  if (error) {
+    showToast(
+      Toast.Style.Failure,
+      "Failed fetching review requests",
+      error instanceof Error ? error.message : String(error)
+    );
+  }
+
+  return (
+    <List isLoading={loading} onSearchTextChange={setSearchText}>
```

Comment:
```suggestion
    <List isLoading={loading}>
```
Do you think it's necessary? Without this it's blazing fast (and then you can remove all debounce - if you really want it, then you should use throttle as [stated here instead](https://developers.raycast.com/api-reference/user-interface/list#list).